lava-v2-jobs-from-api.py

```diff
@@ -58,17 +70,33 @@ def main(args):
     }
 
     print "Working on kernel %s/%s" % (tree, branch)
-    url = urlparse.urljoin(api, ("/build?job=%s&kernel=%s&git_branch=%s&status=PASS&arch=%s" % (tree, git_describe, branch, arch)))
+    url_params = urllib.urlencode({
+        'job': tree,
+        'kernel': git_describe,
+        'git_branch': branch,
+        'status': 'PASS',
+        'arch': arch,
+    })
+    url = urlparse.urljoin(api, 'build?{}'.format(url_params))
     print "Calling KernelCI API: %s" % url
-    response = requests.get(url, headers=headers)
-    if response.status_code != 200:
-        print "Error calling KernelCI API"
-        print response
-        print response.content
-        sys.exit(1)
-    data = json.loads(response.content)
-    builds = data['result']
+
+    builds = None
+    poll_seconds = 10
+    loops = 600 / poll_seconds # time out after 10 minutes
+    loop = 0
+    while not builds and loop < loops:
```

Comment:
Build data right?

If it's possible that the builds arent all in the database by the time this script comes to use it, we need to fix that rather than polling, because how do you know when polling that you've got all the builds? This code would determine 1 result is enough and create jobs for it.

We do many builds per arch so the first one will be in the db well before the last one is finished, maybe we need to know how the longest time the celery cache can be out of date and just wait that long?